### DISCUSSION TITLE
ECAPA-TDNN Model update

### DISCUSSION THREAD

**Discussion by black-mold** [speechbrain/speechbrain#2365] [Feature Request]
### 🚀 The feature

According to [Speaker Verification recipe](https://github.com/speechbrain/speechbrain/tree/develop/recipes/VoxCeleb/SpeakerRec) it has been announced that the weights of the ECAPA-TDNN model trained on VoxCeleb2 are expected to be updated. When will the update take place?

### Solution outline

Reference model

**Comment by mravanelli** [comment]
Hi @black-mold,
in our records, we only have the pretrained models and checkpoints for voxceleb1 + voxceleb2 (which is already shared in the readme file). I cannot find the logs for voxceleb 2 only.
